### DISCUSSION TITLE
Error: Error fetching Hubs Cloud config "Unauthorized"

### DISCUSSION THREAD

**Discussion by smashrockwell** [mozilla/hubs#3630] [Hubs Cloud Questions] [answered]
I’ve forked the Hubs client repository and have it building successfully on my local machine. After running `npm run login`, successfully pointing to my stack’s domain & signing in with an account with admin privileges, I get the following error:

Error: Error fetching Hubs Cloud config "Unauthorized"

**Comment by fredalemany** [comment]
The same problem just now !

Fred

**Reply by smashrockwell** [reply]
You have to use the email address you used when building the stack. Admin Email. Hope that helps

**Comment by fredalemany** [comment]
Yes, the problem and the issu is here !

https://github.com/mozilla/hubs/pull/2340

Thanks

**Comment by robinkwilson** [comment, marked as answer]
I'd double check that the email you're using is an admin. 

Check email is an admin: 
HC Admin Panel > Accounts > Find account by email > check if the email is an admin account. 

If not, just select the "Is Admin" toggle to the right (should be colored).

**Reply by robinkwilson** [reply]
Marking as answered, please "unmark as answered" if you have more information!
